modules/import/csv_file.py

```diff
@@ -23,6 +23,7 @@ class Module(BaseModule):
 
     def __init__(self, *args, **kwargs):
         result = BaseModule.__init__(self, *args, **kwargs)
+        self.__option_prefix = 'CSV_'
```

Comment:
Somewhere it looks like options where switched from lowercase to uppercase. This both pulls out a re-used value and matches the case to what the current recon-ng implementation is.